src/Andoromeda.Kyubey.Portal/Views/Token/_TokenInfoBox.cshtml

```diff
@@ -16,7 +17,7 @@
                     </div>
                     <div class="token-incubation-description-outer">
                         <h3>@Model.Id</h3>
-                        <div style="">@Model.Description</div>
+                        <div>@ViewBag.TokenDescription</div>
```

Comment:
Did you rendered markdown? Please use @Html.Marked(ViewBag.TokenDescription)

Reply:
it's not markdwon text